src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useContext } from 'react';
+import { PersonItem } from '../PersonItem/PersonItem';
+import { PeopleContext } from '../../store/PeopleContext';
+
+export const PeopleList: React.FC = () => {
+  const { peoples, errorMessage } = useContext(PeopleContext);
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      { errorMessage === '' && (
+        <>
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Sex</th>
+              <th>Born</th>
+              <th>Died</th>
+              <th>Mother</th>
+              <th>Father</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {peoples.map((person) => (
+              <PersonItem person={person} key={person.slug} />
+            ))}
+          </tbody>
+        </>
+      )}
+      { peoples.length === 0 && errorMessage === '' && (
```

Comment:
```suggestion
      { !peoples.length && !errorMessage && (
```
pls see [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)